ObservatoryCore/ErrorReporter.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Observatory
+{
+    public static class ErrorReporter
+    {
+        public static void ShowErrorPopup(string title, string message)
+        {
+            if (Avalonia.Application.Current.ApplicationLifetime is Avalonia.Controls.ApplicationLifetimes.IClassicDesktopStyleApplicationLifetime desktop)
+            {
+                var errorMessage = MessageBox.Avalonia.MessageBoxManager
+                .GetMessageBoxStandardWindow(new MessageBox.Avalonia.DTO.MessageBoxStandardParams
```

Comment:
Is this a modal message box?

Reply:
It is, just copied the existing error window verbatim, will chage to non-modal before release. Thanks for the reminder.